src/definitions/indicatorDefinitions.js

```diff
@@ -2,22 +2,47 @@ export const DIAMOND_MODEL = ["adversary", "capability", "infrastructure", "vict
 export const QUERY_TYPES = ["opensearch", "osquery", "sql", "splunk", "censys", "shodan"];
 
 export const INDICATOR_TYPES = [
+  {
+    name: "Forensic artifact",
+    type: "forensicartifact",
+    fields: [
+      { field: "created", type: "date", label: "Created", displayList: true, editable: false, width: "170px" },
+      { field: "name", type: "text", label: "Name", displayList: true, editable: true },
+      { field: "pattern", type: "code", label: "Pattern", displayList: false, editable: true },
+      { field: "relevant_tags", type: "list", label: "Relevant tags", displayList: true, editable: true },
+      { field: "aliases", type: "list", label: "Aliases", displayList: false, editable: true },
+      { field: "supported_os", type: "list", label: "Supported OS", displayList: true, editable: true, width: "140px" },
+      { field: "location", type: "text", label: "Location", displayList: false, editable: true },
+      {
+        field: "diamond",
+        type: "option",
+        choices: DIAMOND_MODEL,
+        label: "Diamond model",
+        displayList: true,
+        editable: true,
+        width: "160px"
+      },
+      { field: "description", type: "longtext", label: "Description", displayList: false, editable: true }
+    ],
+    icon: "mdi-script-text-outline"
+  },
   {
     name: "Regular expression",
     type: "regex",
     fields: [
-      { field: "created", type: "date", label: "Created", displayList: true, editable: false, width: "200px" },
+      { field: "created", type: "date", label: "Created", displayList: true, editable: false, width: "170px" },
       { field: "name", type: "text", label: "Name", displayList: true, editable: true },
       { field: "pattern", type: "code", label: "Pattern", displayList: false, editable: true },
       { field: "relevant_tags", type: "list", label: "Relevant tags", displayList: true, editable: true },
-      { field: "location", type: "text", label: "Location", displayList: true, editable: true },
+      { field: "location", type: "text", label: "Location", displayList: false, editable: true },
```

Comment:
Do you think it's not worth displaying location?

Reply:
https://github.com/yeti-platform/yeti-feeds-frontend/issues/147 for good measure